soccer/planning/TrajectoryUtils.cpp

```diff
@@ -79,11 +75,11 @@ bool TrajectoryHitsDynamic(const Trajectory& trajectory,
         return false;
     }
 
-    // Limit iterations to 400. This will continue to operate at dt = 0.05 until
-    // we hit a 20 second trajectory. If our trajectory is longer than that,
+    // Limit iterations to 100. This will continue to operate at dt = 0.05 until
+    // we hit a 5 second trajectory. If our trajectory is longer than that,
     // something is probably wrong, but we'll still handle it (just scale dt
     // accordingly)
-    constexpr int max_iterations = 400;
+    constexpr int max_iterations = 100;
```

Comment:
I feel like this should be a parameter somewhere? Maybe todo or something?

Reply:
Eh, this isn't really something we'll ever want to change...it doesn't affect behavior, it should basically be "as low as possible without affecting behavior". It can be a param though, sure.